### ISSUE TITLE
Make configuration.env easier to get right

### ISSUE THREAD

**Issue by hammer** [hammerlab/stratotemplate#22]
- [ ] We should indicate which environment variables need to be uncommented
- [ ] `PREFIX` and `TOKEN` could be randomly generated
- [ ] On the `GClouder` image, `GCLOUD_HOST` could be populated for the user

**Comment by hammer** [comment]
Although since these lines are commented out, maybe I don't need to edit them?

**Comment by ihodes** [comment]
Could use better commenting, but `PREFIX` is typically something descriptive (rather than random) like "hammer-strato", as it'll show up as the prefix for the cluster name in the GCP instance list. 

`TOKEN` could certainly be autogenerated with a Make file or something.

`GCLOUD_HOST` is the specific gcloud dev node that'll be the jumping-off point for the strato cluster, so this must be filled in by the user specifically.

**Comment by hammer** [comment]
> GCLOUD_HOST is the specific gcloud dev node that'll be the jumping-off point for the strato cluster, so this must be filled in by the user specifically.

Well I just ran `whomai` and `hostname` on my GCE image and put the results where the placeholders were located. Why couldn't a script do that?

**Comment by ihodes** [comment]
Good point; forgot the names are the same! Definitely nice to generated as much of the config as possible.

**Comment by smondet** [comment]
Yes I thought of generating it but, feared that people may deviate a little bit from the instructions and then break it very easily.

**Comment by arahuja** [comment]
> Well I just ran whomai and hostname on my GCE image and put the results where the placeholders were located. Why couldn't a script do that?

Also depends on where this is done. I also thought this was the case but would be different when running inside the docker container

**Comment by hammer** [comment]
> Also depends on where this is done.

Good point, modified it to indicate where that simplification could be performed
